order/buyer/receive/simple-order-event.md

```diff
@@ -0,0 +1,167 @@
+
+* `orderId`: the Tradecloud order identifier
+* `buyerOrder`: the buyer part of the order, see [Buyer order](#buyer-order)
+* `supplierOrder`: the supplier part of the order, see [Supplier order](#supplier-order)
+* `indicators.deliveryOverdue` is true when at least one order line is overdue.
+* `status.processStatus`: is the aggregate of all lines' [Process statuses](#process-status).
+* `status.logisticsStatus`: is the aggregate of all lines' [Logistics statuses](#logistics-status).
+* `version`: the Tradecloud order version number
+* `eventDates`: some key order event date/times
+* `meta`: meta information, including source and trace info, about this messsage
+* `lastUpdatedAt`: is the latest date time the order has been changed.
+
+### Buyer order
+
+`buyerOrder` is mostly an echo of your order fields as explained in [Issue a new order](../issue/#order-body-json-objects)
+
+* `supplierAccountNumber`: the supplier account number as known in your ERP system
+
+### Supplier order
+
+`supplierOrder` contains the supplier order fields:
+
+* `companyId`: the supplier's Tradecloud company identifier.
+* `buyerAccountNumber`: your account number as known in the supplier's ERP system.
+* `description`: a free format additional description of this order by the supplier.
+* `contact`: the supplier employee responsible for this order.
+* `properties`: are key-value based custom fields, added by the supplier.
+* `notes`: are simple custom fields, added by the supplier.
+* `documents`: contain meta data and link of attached documents by the supplier.
+
+{% page-ref page="download-document.md" %}
+
+### Status
+
+#### Process status
+
+{% hint style="info" %}
+Order and line **process** status is one of:
+
+* `Issued`: \(re\)issued by the buyer.
+* `InProgress`: under negotiation between buyer and supplier
+* `Confirmed`: agreed between buyer and supplier
+* `Rejected`: rejected by supplier
+* `Completed`: completed at the buyer
+* `Cancelled`: cancelled by the buyer
+  {% endhint %}
+
+#### Logistics status
+
+{% hint style="info" %}
+Order, line and delivery line **logistics** status is one of:
+
+* `Open`: no or partial quantity Produced, ReadyToShip, Shipped or Delivered
+* `Produced`: full quantity produced by the supplier
+* `ReadyToShip`: full quantity ready to be shipped by the supplier
+* `Shipped`: full quantity shipped by the supplier
+* `Delivered`: full quantity delivered at the buyer
+* `Cancelled`: cancelled by the buyer
+  {% endhint %}
+
+## Order lines
+
+`lines` contains one or more order lines:
+
+* `id`: the Tradecloud line identifier
+* `buyerLine`: the buyer part of the order line, see [Buyer line](#buyer-line).
+* `supplierLine`: the supplier part of the order line, see [Supplier line](#supplier-line).
+* `confirmedLine`: the order line as agreed between buyer and supplier. Advised is to use the `statusLine.deliverySchedule` and `statusLine.prices` instead.
```

Comment:
Isn't this dependend on the situation?

Reply:
It is a general advise, but I removed it here because there is indeed no context.